content/intro-to-storybook/angular/ja/conclusion.md

```diff
@@ -20,13 +20,13 @@ Storybook は React、React Native、Vue、Angular、Svelte、その他のフレ
 
 - [**楽しい Storybook のワークフロー**](https://www.chromatic.com/blog/the-delightful-storybook-workflow)では Squarespace や、メジャーリーグサッカー、ディスカバリーネットワーク、 Apollo GraphQL といった効率の良いチームにおけるワークフローのベストプラクティスを紹介しています。
```

Comment:
@nagashimam Aside from the Official Storybook documentation text and link, which is great, can you adjust the remainder links and text to match the ones we have in the English version [here](https://github.com/chromaui/learnstorybook.com/blob/master/content/intro-to-storybook/angular/en/conclusion.md).